src/test/scala/org/broadinstitute/dsde/workbench/sam/google/GoogleExtensionSpec.scala

```diff
@@ -579,10 +579,10 @@ class GoogleExtensionSpec(_system: ActorSystem) extends TestKit(_system) with An
     val googleExtensions = new GoogleExtensions(TestSupport.fakeDistributedLock, mockDirectoryDAO, mock[RegistrationDAO](RETURNS_SMART_NULLS), null, mockGoogleDirectoryDAO, null, null, null, null, null, null, null, null, null, googleServicesConfig, null, configResourceTypes)
 
     val allUsersGroup = BasicWorkbenchGroup(CloudExtensions.allUsersGroupName, Set.empty, WorkbenchEmail(s"TEST_ALL_USERS_GROUP@test.firecloud.org"))
-    val allUsersGroupMatcher = new ArgumentMatcher[BasicWorkbenchGroup] {
-      override def matches(group: BasicWorkbenchGroup): Boolean = group.id == allUsersGroup.id
+    val allUsersGroupNameMatcher = new ArgumentMatcher[WorkbenchGroupName] {
```

Comment:
my intellij is taking forever to load up so I'm just going to ask here -- why the change from `WorkbenchGroupName` to `BasicWorkbenchGroup`?

Reply:
Since the ` getOrCreateAllUsersGroup` method now checks for the existence of the all-users group and only tries to create it if it doesn't exist, I needed to change the mocked method to provide a response for the `loadGroup` method instead of the `createGroup` method, and that has a different signature.

Reply:
oh right, that makes sense! thanks for the explanation!